src/gemini/index.ts

```diff
@@ -0,0 +1,80 @@
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+import * as fs from "fs";
+import * as path from "path";
+import { promptOnce } from "../prompt";
+import { execSync } from "child_process";
+import * as clc from "colorette";
+
+const { GoogleGenerativeAI } = require("@google/generative-ai");
+
+// TODO(christhompson): Do we have an endpoint that's open and doesn't require a project w/ billing?
+// Estimated QPS is around 1.
+// TODO(christhompson): Add project ID for this.
+// TODO(christhompson): Add preamble information about command flags.
+// TODO(christhompson): Figure out how to test this.
+const generationConfig = {
+  maxOutputTokens: 200,
+  temperature: 0.9,
```

Comment:
Temp seems a little spicy here -- Did you experiment with a different values?  My assumption is that if the temp needs to be this high, then we're likely not prompting enough.